### ISSUE TITLE
Dependabot Alert for loader-utils <1.4.2 on nrwl 13.10.x

### ISSUE THREAD

**Issue by mgithubmessier** [nrwl/nx#19025]
### Current Behavior

Dependabot Alets go off when using nx@13.10.x
I have submitted [PR](https://github.com/nrwl/nx/pull/19024) with what I believe should resolve this

### Expected Behavior

There should be no critical dependabot alerts.

### Steps to Reproduce

1. Install nrwl/web@13.10.x
2. Look at the yarn.lock
3. see that loader-utils is fixed at version 1.2.3

### Nx Report

```shell
Node : 16.18.0
   OS   : darwin x64
   yarn : 1.22.17
   
   nx : 13.10.0
   @nrwl/angular : Not Found
   @nrwl/cypress : 13.10.0
   @nrwl/detox : Not Found
   @nrwl/devkit : 13.10.0
   @nrwl/eslint-plugin-nx : 13.10.0
   @nrwl/express : Not Found
   @nrwl/jest : 13.10.0
   @nrwl/js : 13.10.0
   @nrwl/linter : 13.10.0
   @nrwl/nest : Not Found
   @nrwl/next : 13.10.0
   @nrwl/node : Not Found
   @nrwl/nx-cloud : 13.2.2
   @nrwl/nx-plugin : Not Found
   @nrwl/react : 13.10.0
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 13.10.0
   @nrwl/web : 13.10.0
   @nrwl/workspace : 13.10.0
   typescript : 4.6.2
   rxjs : 6.6.7
   ---------------------------------------
   Community plugins:
   	 @ngrx/component-store: 13.0.2
   	 @ngrx/effects: 13.0.2
   	 @ngrx/entity: 13.0.2
   	 @ngrx/router-store: 13.0.2
   	 @ngrx/schematics: 13.0.2
   	 @ngrx/store: 13.0.2
   	 @ngrx/store-devtools: 13.0.2
   	 @storybook/angular: 6.4.19
```

### Operating System

- [X] macOS
- [ ] Linux
- [ ] Windows
- [ ] Other (Please specify)

**Comment by FrozenPandaz** [comment]
Sorry, Nx 13.x is over a year old now so we are not supporting it anymore. This issue has been solved in 15.x. Could you update?

We could provide a patch for 14.x if necessary.

**Comment by FrozenPandaz** [comment]
I released `14.8.9` with a fix for this.
